ngraph/src/ngraph/opsets/opset0_tbl.hpp

```diff
@@ -102,6 +102,7 @@ NGRAPH_OP(EmbeddingLookup, ngraph::op)
 NGRAPH_OP(Equal, ngraph::op)
 NGRAPH_OP(Erf, ngraph::op)
 NGRAPH_OP(Exp, ngraph::op)
+NGRAPH_OP(ExtractImagePatches, ngraph::op)
```

Comment:
Removed this line. The ExtractImagePatches is not a part of opset0.

Reply:
Fixed